spring-integration-mongodb/src/main/java/org/springframework/integration/mongodb/support/MongoDbMessageBytesConverter.java

```diff
@@ -32,14 +32,17 @@
 
 /**
  * A {@link GenericConverter} implementation to convert {@link Message} to
- * serialized {@link byte[]} to store {@link Message} to the MongoDB.
- * And vice versa - to convert {@link byte[]} from the MongoDB to the {@link Message}.
+ * serialized {@code byte[]} to store {@link Message} to the MongoDB.
+ * And vice versa - to convert {@code byte[]} from the MongoDB to the {@link Message}.
 
  * @author Artem Bilan
  * @author Gary Russell
  * @since 4.2.10
  * @deprecated since 5.0 in favor of {@link MessageToBinaryConverter} and {@link BinaryToMessageConverter}
```

Comment:
Maybe we can remove this class now?

I think that is a good catch for wrong JavaDocs - I don't see reason to add comment for that :smile: 

But thanks anyway!